File: python/cash.py
from cs50 import get_float
ch = 0
while ch <= 0:
    ch = int(get_float("Введите сдачу: ")*100)

t25 = 0
t10 = 0
t5 = 0
t1 = 0
while ch >= 25:

    ch = ch - 25
    t25+=1

while ch >= 10:
    ch = ch - 10
    t10+=1

while ch >= 5:
    ch = ch - 5
    t5+=1

while ch >= 1:
    ch = ch - 1
    t1+=1

print(t25+t10+t5+t1)

File: pset0/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    float cash;
    for (;;){
        cash = get_float("Введите сдачу\n");
        if (cash <= 0) {
            printf("Вы ввели неверное число\n");
            continue;
        }
        else {
        break;
        }
    break;
    }
    int c25, c10, c5, c1;
    c25 = cash*100 / 25;
    c10 = (cash*100 - c25*25) / 10;
    c5 = (cash*100 - c25*25 - c10*10) / 5;
    c1 = cash*100 - c25*25 - c10*10 - c5*5;
    printf("Сдача в монетах 25 центов (шт): %i\n", c25);
    printf("Сдача в монетах 10 центов (шт): %i\n", c10);
    printf("Сдача в монетах 5 центов (шт): %i\n", c5);
    printf("Сдача в монетах 1 цент (шт): %i\n", c1);
}